.github/workflows/release-homebrew.yaml

```diff
@@ -85,8 +88,37 @@ jobs:
           brew bump-cask-pr --write-only -f --version=$FINCH_VERSION finch
       - name: Silently install
         run: |
+          export HOMEBREW_NO_INSTALL_FROM_API=1
           cd $(brew --repo homebrew/cask)
           brew install --cask ./Casks/finch.rb
+      - name: Build project
+        run: |
+          brew install go lz4 automake autoconf libtool
+          export PATH="/opt/homebrew/opt/libtool/libexec/gnubin:$PATH"
+          which libtool
+          make
+      - name: Multiple instances of Finch test
+        run: |
+          # start two Finch VM instances
+          ./_output/bin/finch vm init
+          finch vm init
+          # start a container in each VM instance
+          ./_output/bin/finch pull alpine
+          finch pull alpine
+          ./_output/bin/finch run --name test-ctr1 alpine
+          finch run --name test-ctr2 alpine
+          # clean up the VMs
+          ./_output/bin/finch vm stop && ./_output/bin/finch vm remove
+          finch vm stop && finch vm remove
+      - name: Clean up multiple instance test
+        run: |
+          sudo rm -rf ./_output
+          export HOMEBREW_NO_INSTALL_FROM_API=1
+          cd $(brew --repo homebrew/cask)
+          # Need to resintall because there were errors on arm64 11.7 and arm64 12.6 hosts after running multiple instances tests,
```

Comment:
```suggestion
          # Need to reinstall because there were errors on arm64 11.7 and arm64 12.6 hosts after running multiple instances tests,
```